File: woodroid/src/main/java/me/gilo/woodroid/models/Line_item.java
package me.gilo.woodroid.models;

public class Line_item {
    private int product_id;
    private int quantity;
    private String variations;

    public Line_item(int product_id, int quantity, String variations) {
        this.product_id = product_id;
        this.quantity = quantity;
        this.variations = variations;
    }

    public Line_item(int product_id, int quantity) {
        this.product_id = product_id;
        this.quantity = quantity;
    }

    public int getProduct_id() {
        return product_id;
    }

    public void setProduct_id(int product_id) {
        this.product_id = product_id;
    }

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }

    public String getVariations() {
        return variations;
    }

    public void setVariations(String variations) {
        this.variations = variations;
    }
}

File: woodroid/src/main/java/me/gilo/woodroid/models/LineItem.kt
package me.gilo.woodroid.models

import com.google.gson.annotations.SerializedName

import java.util.ArrayList


class LineItem {

    var subtotal: String
    @SerializedName("subtotal_tax")
    var subtotalTax: String
    var total: String
    var totalTax: String
    var price: String
    var quantity: Int = 0
    var taxClass: Any
    var name: String

    @SerializedName("product_id")
    var productId: Int = 0

    var sku: String
    var variations: String
    var meta: List<Metum> = ArrayList()
}
